### PR TITLE
cleanup for sl4

### PR DESCRIPTION
**Pull request by braver**
WIP: I need to check what those selectors would translate to.

### CODE REVIEW

File: linter.py [hunk 1]
```diff
@@ -1,41 +1,17 @@
...
     error_stream = util.STREAM_STDOUT
     regex = (
         r'^(?P<line>\d+):(?P<col>\d+) '
         r'(?:(?P<error>Error)|(?P<warning>Warning)) '
         r'(?P<message>.+)'
     )
-
-    config_file = ('--localrc', 'chktexrc')
     defaults = {
+        'selector': 'text.tex.latex',
```
Comment (kaste) [inline linter.py]: Isn't this just `text.tex.latex  - meta.block.parameters.knitr - source.r.embedded.knitr`?

Otherwise LGTM

Comment (braver) [reply linter.py]: Hm, good point. Could it be...

```
text.tex.latex, text.tex.latex.knitr - meta.block.parameters.knitr - source.r.embedded.knitr
```

❓ 

Damn, now I need to go and find out how this chktex thing works. Or guess and see if it breaks for someone.

Comment (kaste) [reply linter.py]: No, then it will match the 'wider'(?) region. E.g. if you select `source, source.python - comment` it will select the whole file.

Comment (braver) [reply linter.py]: yeah, makes sense. Let's just go with your selector.

**Review by kaste — APPROVED** [review]

**Pull request merged**